src/irmin-pack/inode_intf.ml

```diff
@@ -78,7 +78,7 @@ module type INTER = sig
     val save : add:(hash -> Elt.t -> unit) -> mem:(hash -> bool) -> t -> unit
     val hash : t -> hash
     val stable : t -> bool
-    val length : t -> int
+    val length : t -> int64
```

Comment:
why do you need an `int64` here?

Reply:
`irmin_git` exposes a `length : t -> int64` function. I flipped a coin and decided to use int64 everywhere

Reply:
other things will break if we ever have more than 1 billion entries. Keeping `int` here is better I think (especially since `list` takes an `int` as `offset`).

Reply:
I have converted everything back to `int`